cmake/CMakeListsNrnMech.cmake

```diff
@@ -30,16 +30,14 @@ foreach(link_lib ${NRN_LINK_LIBS})
 
   get_filename_component(dir_path ${link_lib} DIRECTORY)
   if(TARGET ${link_lib})
-    message(NOTICE "Using Target in compiling and linking, you should take care of it if
-                    it fail miserabily (CODE: 1234567890)")
     get_property(link_flag TARGET ${link_lib} PROPERTY INTERFACE_LINK_LIBRARIES)
-    string(APPEND NRN_LINK_DEFS ${link_flag})
+    set(description "Extracting link flags from target '${link_lib}', beware that this can be fragile.")
```

Comment:
I can see a face of novice user curiously looking at this and not knowing what to do 😇